helm/metadig-controller/config/metadig.properties

```diff
@@ -4,21 +4,27 @@
 DataONE.authToken = not-set in config
 CN.subjectId = CN=urn:node:CN,DC=dataone,DC=org
 CN.serviceUrl = https://cn.dataone.org/cn
-# The subjectId is used during harvesting of system metadata and metadata from this DataONE member node. THis is
+# The subjectId is used during harvesting of system metadata and metadata from this DataONE member node. This is
 # required in order to read non-public content.
 KNB.subjectId = CN=urn:node:KNB,DC=dataone,DC=org
 KNB.serviceUrl = https://knb.ecoinformatics.org/knb/d1/mn
 ARCTIC.subjectId = CN=urn:node:ARCTIC,DC=dataone,DC=org
 ARCTIC.serviceUrl = https://arcticdata.io/metacat/d1/mn
 ESS_DIVE.subjectId = CN=urn:node:ESS-DIVE,DC=dataone,DC=org
 ESS_DIVE.serviceUrl = https://data.ess-dive.lbl.gov/catalog/d1/mn
+CA_OPC.subjectId = CN=urn:node:CA_OPC,DC=dataone,DC=org
+CA_OPC.serviceUrl = https://opc.dataone.org/metacat/d1/mn
+CERP_SFWMD.subjectId = CN=urn:node:CERP_SFWMD,DC=dataone,DC=org
+CERP_SFWMD.serviceUrl = https://cerp-sfwmd.dataone.org/metacat/d1/mn
+SCTLD.subjectId = CN=urn:node:SCTLD,DC=dataone,DC=org
+SCTLD.serviceUrl = https://sctld.dataone.org/metacat/d1/mn
+DRP.subjectId = urn:node:DRP
+DRP.serviceUrl = https://drp.dataone.org/metacat/d1/mn
```

Comment:
Having to have node-specific configuration params does not seem scalable to me. Can't this information be stored elsewhere, and/or looked up? We should discuss how this would work with 50 new member repos.

Reply:
Yeah this is on my mind as well - I'll create a ticket for it